core/templates/pages/diagnostic-test-player-page/diagnostic-test-player.component.ts

```diff
@@ -78,10 +101,17 @@ export class DiagnosticTestPlayerComponent implements OnInit {
     );
 
     this.getProgressText();
+
     this.classroomBackendApiService
-      .getClassroomIdAsync(this.classroomUrlFragment)
-      .then(classroomId => {
-        this.classroomId = classroomId;
+      .fetchClassroomDataAsync(this.classroomUrlFragment)
```

Comment:
Would it be possible to keep using getClassroomIdAsync? Since it has the topicIdToPrerequisites it would require less changes.

I don't insist, just wondering

Reply:
Yes we can do that.

Reply:
Done.